### DISCUSSION TITLE
v3.0.6-beta

### DISCUSSION THREAD

**Discussion by mrghozzi** [mrghozzi/myads#40] [Announcements]
**Fix**: security bugs.
**Added**: PTS History.
**Fix**: Syntax error in language files.
**Fix**: Error in site directory notifications.
**Added**: Button to report ads.
**Improve** the template /follow.php
**Change**: Twitter logo
**Update**: Font Awesome library to version v6.4.2
**Improvement**: The appearance of forum topics in the portal

**Comment by alaa-eg** [comment]
اقتراحات للتحديث القادم

- حذف الاختيارات المكرره
https://ibb.co/N9kTD0c

- عند اختبار اكتابة من اليمين إلى اليسار في صندوق الكتابة ، لا يظهر في الوصف
https://ibb.co/cYmVQ4d

**Reply by mrghozzi** [reply]
مرحبا.

-  بالنسبة للخيارات المكررة في المتجر
 لديك خطأ في قاعدة البيانت, يبدو أنك عند تثبيت الإصدار السابق v2.3.3 قمت بتشغيل صفحة التحديث مرتين.
للإصلاح إفتح قاعدة البيانات `phpmyadmin` و بإبحث في جدول `options` عن `o_type` التالية:

1.السكربتات `scriptcat`
2. الأقسام `storecat`
و إحدف الخيارات المكررة
مثال
https://www.adstn.gq/upload/1700425366_u.png

1. ``SELECT * FROM `options` WHERE `o_type` LIKE 'scriptcat' ``

2. ``SELECT * FROM `options` WHERE `o_type` LIKE 'storecat' ``

- بالنسبة لصندوق الكتابة من اليمين لليسار إتبع المثال التالي
https://www.adstn.gq/upload/1700425692_x.png
https://www.adstn.gq/upload/1700426516_q.png

**Reply by alaa-eg** [reply]
**شكراً جزيلاً - تم حل المشكلة**

**Comment by alaa-eg** [comment]
- هذه الروابط لا تعمل في لوحة تحكم المدير

```
/admincp?state&ty=banner&st=1

/admincp?state&ty=link&st=1
```


https://ibb.co/7tQ6z6T

**Reply by mrghozzi** [reply]
جرب إعادة رفع الملفين
`/state.php`
`/templates/_panel/state.php`
إن لم ينجح الأمر سأبحث عن أصل المشكل

**Reply by alaa-eg** [reply]
تم إعادة الرفع ، نفس المشكلة

**Reply by mrghozzi** [reply]
سأبحث عن سبب المشكل
تم الإضافة في جدول الأعمال https://github.com/mrghozzi/myads/issues/52

**Comment by alaa-eg** [comment]
- بنر 300*250 خارج الودجات في الشاشات الكبيرة
https://ibb.co/dDVxwGQ

**Reply by mrghozzi** [reply]
في الودجات إستخدم البنر ''المتجاوبة''  لأن مقاس 250/300 غير متوافق مع مقاس الودجات

**Reply by alaa-eg** [reply]
البنر المتجاوب يظهر بانر صغير جداً

قمت باضافة بنر 160*600 فهو أفضل ومتناسب

**Comment by alaa-eg** [comment]
- اسم المنتدى الطويل لا يظهر ضمن المكان المحدد
https://ibb.co/Fs5nJxs

الأفضل حذف     height: 86px; في 

```
@media screen and (max-width: 960px)
.section-banner {
    height: 86px;
    min-height: auto;
    padding: 18px 28px 0 108px;
}
```

**Reply by mrghozzi** [reply]
تم إضافة المقترح لجدول الأعمال https://github.com/mrghozzi/myads/issues/50

**Comment by alaa-eg** [comment]
- اقتراح بتوسيط الأقسام @ جعل ترتيب الأقسام الفرعية بالترتيب من الأول للآخر ( حالياً الترتيب بالعكس) @ إضافة إمكانية الضغط على الأقسام الفرعية والدخول لها مباشرة
https://ibb.co/n35nsy1

- حذف مكان "الأقسام" عندما لا يكون هنا أقسام فرعية
https://ibb.co/4SLHkzx

- إمكانية ترتيب أقسام المنتديات forum
- توسيط عنوان الموضوع والموضوع حتى يناسب مع كل اللغات
https://ibb.co/GHjVvvw

- توسيط في دليل المواقع ليتناسب مع جميع اللغات
https://ibb.co/s9Q0w48

- تبسيط عرض المواقع الدليل بدون صورة الموقع مثل دليل نواحي
https://ibb.co/3FRp30F
وتكون الصورة عند الدخول لصفحة معلومات الموقع
https://www.nwahy.com/free/dir/site-25815.html

**Reply by mrghozzi** [reply]
تم إضافة المقترحات لجدول الأعمال https://github.com/mrghozzi/myads/issues/51

**Reply by mrghozzi** [reply]
#57 #56 #55
